javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,18 @@
+var fullName = 'Miori Imai';
+console.log('value of fullName: ', fullName);
+var isCool = true;
+console.log('value of isCool: ', isCool);
+var totalPets = 2;
+console.log('value of totalPets: ', totalPets);
+
+console.log('typeof fullName:', typeof 'Miori Imai');
+console.log('typeof isCool:', typeof true);
+console.log('typeof totalPets:', typeof 2);
```

Comment:
for the `typeof` console logs, you want to use the variables, not the values themselves
console.log("typeof fullName", typeof fullName)